src/main/kotlin/no/nav/klage/oppgave/api/view/KodeverkResponse.kt

```diff
@@ -1,23 +1,68 @@
 package no.nav.klage.oppgave.api.view
 
 import no.nav.klage.kodeverk.*
-import no.nav.klage.kodeverk.hjemmel.Hjemmel
-import no.nav.klage.kodeverk.hjemmel.hjemlerPerTema
-import no.nav.klage.kodeverk.hjemmel.hjemlerPerYtelse
+import no.nav.klage.kodeverk.hjemmel.*
 
 data class KodeverkResponse(
-    val hjemmel: List<Kode> = Hjemmel.values().asList().toDto(),
```

Comment:
Jeg er ikke 100% sikker på om vi kan fjerne denne? Det fordrer at vi eksponerer et tilsvarende endepunkt i kabal-search, for frontend trenger vel disse fremdeles?

Reply:
Den er ikke fjernet. Lagt de som skal ryddes opp litt lenger ned.